code/src/index.css

```diff
@@ -1,13 +1,234 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
+    'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #191414;
+  margin: 5px;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+}
+
+.container {
+  width: 300px;
+  margin: 0 auto;
+  padding: 40px 0;
+}
+
+.menu {
+  position: fixed;
+  top: 0;
+  right: 10px;
+  width: 25px;
+  filter: invert();
+  z-index: 2;
+  margin: 20px 10px;
+}
+
+.menu:hover {
+  width: 30px;
+  transition: 0.3s;
+}
+
+.title {
+  margin: 10px 5px;
+  color: #ffffff;
+  font-size: 25px;
+  border-bottom: 1px solid rgb(55, 55, 55);
+  padding-bottom: 5px;
+}
+
+.album-wrapper {
+  display: flex;
+  flex-direction: row;
+}
+
+.album-grid + .title:last-of-type {
+  padding-top: 50px;
+}
+
+.album-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 20px;
+  row-gap: 40px;
+}
+
+.album-container {
+  justify-self: center;
+}
+
+/* ALBUM COVER */
```

Comment:
We guess that this is the styling that your first try to add in separate css components?

Nice to also use explanatory comments for these different styling elements and make the code more readable.